config/300-gateway.yaml

```diff
@@ -36,7 +36,7 @@ spec:
             - --log-level info
           command:
             - /usr/local/bin/envoy
-          image: docker.io/envoyproxy/envoy:v1.16-latest
+          image: docker.io/envoyproxy/envoy:v1.17-latest
```

Comment:
I'd leave the default at 1.16 for now, to make sure upgrade cases will go just fine. We can test 1.17 in addition to that though via the gateway list above.

Reply:
bad news.. 1.16 complains about the config due to a bug in envoy.. 1.17 works, I'm just playing around, trying to get the test passing, and then I will try to make it work on 1.16 and 1.14 properly

Reply:
I need to dig a little bit more into that issue: https://github.com/envoyproxy/envoy/issues/14272

Reply:
Gotcha, okay. Might make sense to slap a `[WIP]` on the PR.